packages/lb-annotation/src/core/pointCloud/store/index.ts

```diff
@@ -191,18 +206,23 @@ class PointCloudStore {
            * TODO: Has visible.
            *
            */
-          if (this.cloudData.get(key).visible === false || this.segmentMode === EPointCloudSegmentMode.Remove) {
-            if (this.segmentMode === EPointCloudSegmentMode.Add) {
-              this.cloudData.get(key).visible = true;
-            }
-            if (this.segmentMode === EPointCloudSegmentMode.Remove) {
-              this.cloudData.get(key).visible = false;
+          if (this.segmentMode === EPointCloudSegmentMode.Remove) {
+            this.cloudData.get(key).visible = false;
+            vertices.push(cloudDataArrayLike[i], cloudDataArrayLike[i + 1], cloudDataArrayLike[i + 2]);
+          }
+          if (this.segmentMode === EPointCloudSegmentMode.Add) {
+            if (this.cloudData.get(key).visible === true) {
+              covers.push(cloudDataArrayLike[i], cloudDataArrayLike[i + 1], cloudDataArrayLike[i + 2]);
             }
             vertices.push(cloudDataArrayLike[i], cloudDataArrayLike[i + 1], cloudDataArrayLike[i + 2]);
```

Comment:
You can add your coverMode's judgement in this place

Reply:
done